src/cli/index.js

```diff
@@ -2,5 +2,13 @@
  * CLI goes here
  */
 const program = require('commander');
+const packageInfo = require('../../package');
+const startServer = require('../server/httpServer');
 
-console.log('Cli goes here');
+program
+  .version(packageInfo.version)
+  .description(packageInfo.description)
+  .option('-p, --port [type]', 'run the app on the port [3000]', 3000)
```

Comment:
Mmh... I think the `[3000]` port information is overwhelmed since we don't want to run it on 3000 by default